src/core/src/op/util/detection_output_base.cpp

```diff
@@ -338,3 +73,20 @@ bool ov::op::util::DetectionOutputBase::visit_attributes_base(AttributeVisitor&
     visitor.on_attribute("objectness_score", attrs.objectness_score);
     return true;
 }
+
+ov::Dimension DetectionOutputBase::compute_num_classes(const AttributesBase& attrs) {
+    NODE_VALIDATION_CHECK(this,
+                          3 == get_input_size() && get_input_size() == 5,
+                          "A number of arguments must be  equal to 3 or equal to 5. Got  ",
+                          get_input_size());
```

Comment:
It will probably fix failed tests
```suggestion
    const auto input_size = get_input_size();
    NODE_VALIDATION_CHECK(this,
                          input_size ==3 || input_size == 5,
                          "A number of arguments must be  equal to 3 or equal to 5. Got  ",
                          get_input_size());
```

Reply:
Thx for the comments. Applied.